docs/logging/logging.md

```diff
@@ -427,4 +447,230 @@ spec:
     url: http://loki-stack.cattle-logging-system.svc:3100
     extra_labels:
       logOutput: harvester-loki
-```
\ No newline at end of file
+```
+
+## Audit
+
+Harvester collects Kubernetes `audit` and is able to send the `audit` to vairous types of log server.
+
+The policy file to guide `kube-apiserver` is [here] (https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rke2-92-harvester-kube-audit-policy.yaml).
+
+### Audit Definition
+
+In `kubernetes`, the [audit](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/) data is generated by `kube-apiserver` according to defined policy.
+
+```
+...
+Audit policy
+Audit policy defines rules about what events should be recorded and what data they should include. The audit policy object structure is defined in the audit.k8s.io API group. When an event is processed, it's compared against the list of rules in order. The first matching rule sets the audit level of the event. The defined audit levels are:
+
+None - don't log events that match this rule.
+Metadata - log request metadata (requesting user, timestamp, resource, verb, etc.) but not request or response body.
+Request - log event metadata and request body but not response body. This does not apply for non-resource requests.
+RequestResponse - log event metadata, request and response bodies. This does not apply for non-resource requests.
+```
+
+### Audit Log Format
+
+#### Audit Log Format in Kubernetes
+
+Kubernetes apiserver logs audit with following JSON format into a local file.
+
+```
+{
+"kind":"Event",
+"apiVersion":"audit.k8s.io/v1",
+"level":"Metadata",
+"auditID":"13d0bf83-7249-417b-b386-d7fc7c024583",
+"stage":"RequestReceived",
+"requestURI":"/apis/flowcontrol.apiserver.k8s.io/v1beta2/prioritylevelconfigurations?fieldManager=api-priority-and-fairness-config-producer-v1",
+"verb":"create",
+"user":{"username":"system:apiserver","uid":"d311c1fe-2d96-4e54-a01b-5203936e1046","groups":["system:masters"]},
+"sourceIPs":["::1"],
+"userAgent":"kube-apiserver/v1.24.7+rke2r1 (linux/amd64) kubernetes/e6f3597",
+"objectRef":{"resource":"prioritylevelconfigurations",
+"apiGroup":"flowcontrol.apiserver.k8s.io",
+"apiVersion":"v1beta2"},
+"requestReceivedTimestamp":"2022-10-19T18:55:07.244781Z",
+"stageTimestamp":"2022-10-19T18:55:07.244781Z"
+}
+```
+
+#### Audit Log Format before Being Sent to Log Servers
+
+Harvester keeps the `audit` log unchanged before sending it to the log server.
+
+### Audit Log Output/Clusteroutput
```

Comment:
```suggestion
### Audit Log Output/ClusterOutput
```
Do we need the upper case for consistency?

Reply:
Good catch, updated with unified format. thanks.